### ISSUE TITLE
BOSS Profile page

### ISSUE THREAD

**Issue by prathambatra** [coding-blocks/boss#409]
Right now when we click on username from the claims page or leaderboard, we reach to Github profile. we can have a separate page for boss profile. it will show all recent work of that user, claims, and position on the leaderboard. from there we can also go to Github profile.

**Comment by hereisnaman** [comment]
Please list down the features you would like to include.

**Comment by prathambatra** [comment]
Options for BOSS Profile Page : 
User's current rank,
Bounty points,
recent work section (Timeline),
tabs for accepted claims and pending claims,
link to Github profile,
link to admin dashboard (if admin)

**Comment by hereisnaman** [comment]
> Options for BOSS Profile Page :
> User's current rank,
> Bounty points,
> recent work section (Timeline),
> tabs for accepted claims and pending claims,
> link to Github profile,
> link to admin dashboard (if admin)

What about stats from previous BOSS seasons?

**Comment by prathambatra** [comment]
> > Options for BOSS Profile Page :
> > User's current rank,
> > Bounty points,
> > recent work section (Timeline),
> > tabs for accepted claims and pending claims,
> > link to Github profile,
> > link to admin dashboard (if admin)
> 
> What about stats from previous BOSS seasons?

yes we will add two options in stat current year and previous year

**Comment by hereisnaman** [comment]
> > > Options for BOSS Profile Page :
> > > User's current rank,
> > > Bounty points,
> > > recent work section (Timeline),
> > > tabs for accepted claims and pending claims,
> > > link to Github profile,
> > > link to admin dashboard (if admin)
> > 
> > 
> > What about stats from previous BOSS seasons?
> 
> yes we will add two options in stats current year and previous year

please share some wireframes.

**Comment by prathambatra** [comment]
![rsz_screenshot_from_2020-06-26_02-30-05](https://user-images.githubusercontent.com/29707062/85795024-36efa680-b755-11ea-94e4-67ec03303353.png)

**Comment by prathambatra** [comment]
@hereisnaman

**Comment by hereisnaman** [comment]
> @hereisnaman

Go ahead.

**Comment by prathambatra** [comment]
Can you assign bounties? @hereisnaman

**Comment by prathambatra** [comment]
@hereisnaman

**Comment by deprov447** [comment]
@prathambatra . I would like to work on this issue if still open and unassigned

**Comment by prathambatra** [comment]
@deprov447 anyone can try here. nothing is assigned. but i am already working and will make PR soon.

**Comment by coderrsid** [comment]
@hereisnaman  @prathambatra can i work on this issue?

**Comment by prathambatra** [comment]
yes please

**Comment by knightmate** [comment]
is this issue open?
